### PR TITLE
Add/automated testing

### PR DESCRIPTION
**Pull request by sambua**
## Proposed changes
- List an overview of the changes included in this PR

## Dependencies
- Update this if the PR requires us to update the project’s dependencies

## Checklist
- [ ] Tested locally
- [ ] Added new unit, integration or regression tests
- [ ] Updated docs

## Demo
- Add before and after screenshots and videos showcasing the changes

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,7 @@
+PLAYWRIGHT_URL=http://localhost
+PLAYWRIGHT_PORT=5000
+PLAYWRIGHT_OS=linux# can also be windows
```
Comment (ntshcalleia) [inline .env:3]:
```suggestion
PLAYWRIGHT_OS=linux # can also be windows
```

File: .env.example
Comment (ntshcalleia) [inline .env.example:1]: Should we remove this since it's empty?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,275 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+    paths-ignore: [ 'demos/**']
```
Comment (ntshcalleia) [inline .github/workflows/ci.yml:6]: Hi @sambua, this breaks some of the requirements from our existing continuous deployment workflow. Can you please remove this section and restore the deleted workflow file? It's better if we QA it before making changes to the existing solution.
```suggestion
```

**Review by ntshcalleia — CHANGES_REQUESTED** [review]

File: .github/actions/build-component/action.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+name: 'Build component'
+description: 'Build component'
+runs:
+  using: "composite"
+  steps:
+    - name: Build
+      shell: bash
+      run: |
+        grunt
```
Comment (ntshcalleia) [inline .github/actions/build-component/action.yml:9]:
```suggestion
        grunt
        grunt
```

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+    #      cancel-in-progress: true
+    steps:
+      # npm install --global http-server # npx http-server ./ [options]
+      # http-server ./ -p 5050
+      # http-server ./ -p 5050 > http.log 2>&1 &
+      - name: Install local web server via http-server package
+        run: |
+          npm install --global http-server
+
+  # act --job 'playground' --container-architecture linux/amd64
```
Comment (ntshcalleia) [inline .github/workflows/ci.yml:148]: @sambua why do we have a job that only installs http-server?
```suggestion
```

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -0,0 +1,275 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+    paths-ignore: [ 'demos/**']
+  pull_request:
+    branches: [ '*' ]
+    types: [ opened, review_requested, edited, reopened ]
+    paths: [ 'src/**', 'tests/**' ]
```
Comment (ntshcalleia) [inline .github/workflows/ci.yml:10]: Let's comment this out for now and test it through `workflow_dispatch`
```suggestion
```

File: .github/actions/unit-test/action.yml
Comment (ntshcalleia) [inline .github/actions/unit-test/action.yml:1]: It doesn't look like this action is being used. Can we delete it?

File: .github/workflows/build.yml
Comment (ntshcalleia) [inline .github/workflows/build.yml:1]: Let's not remove this for now since it works as expected.

**Comment by sambua** [comment]
Hi @ntshcalleia thanks for the review, but actually it's in process job. I've pushed here just for the testing, will push little bit later working option

**Pull request closed without merging**